contributing.md

```diff
@@ -75,11 +75,10 @@ When you are ready to jump into code, use the following workflow to add new comp
 
 ## Getting started
 
-> Make sure to run at least npm 4, which ships with Node 7. You can get the
-[latest Node version from this website](https://nodejs.org/en/), use a tool like
-[nvm](https://github.com/creationix/nvm) (`nvm use` will switch you to a supported version) or
-[nave](https://github.com/isaacs/nave) (`nave auto` does the trick), or just upgrade npm using
-`npm install -g npm@latest`.
+> Backpack development requires Node `8.6.0` and npm `5.3.0`. [Nvm](https://github.com/creationix/nvm) users can run
+`nvm use` to switch to `8.6.0`. [Nave](https://github.com/isaacs/nave) users can use `nave auto`. You can also
+download Node `8.6.0` using [their website](https://nodejs.org/en/). npm `5.3.0` comes with Node `8.6.0` by default. To
+install npm `5.3.0` manually, run `npm install --global npm@5.3.0`.
```

Comment:
This should be  5.4.2 as in the `before_install` command right?

Reply:
oopsies 😇